docs/machine-learning/how-to-guides/load-data-ml-net.md

```diff
@@ -161,7 +161,70 @@ Use the `Load` method to load the data into an [`IDataView`](xref:Microsoft.ML.I
 ```csharp
 IDataView data = loader.Load(dbSource);
 ```
+## Load data from images files
 
+Create a model that hold image path and a lebel . `ImagePath` is the absolute path of the image in data source directory. `Label` is the class or category of the actual image file..
+
+```csharp
+ public class ImageData
+        {
+            [LoadColumn(0)]
+            public string ImagePath;
+
+            [LoadColumn(1)]
+            public string Label;
+        }
+```
+## Load the Image Data from input directory.
```

Comment:
I'm not sure this needs its own section. It's still part of loading images from files.